src/api/v1/apps.ts

```diff
@@ -71,13 +72,17 @@ app.post("/", async (c) => {
     form = result.data;
   } else {
     const formData = await c.req.parseBody();
+    // FIXME: this currently parses without a body
     const result = await applicationSchema.safeParseAsync(formData);
     if (!result.success) {
       logger.debug("Invalid request: {error}", { error: result.error });
       return c.json({ error: "Invalid request", zod_error: result.error }, 400);
     }
     form = result.data;
   }
+
+  console.log({ form });
```

Comment:
I think it's left by mistake?

Reply:
Nope, deliberate. Was debugging that potential bug with empty client body.